apps/web/components/v2/eventtype/AvailabilityTab.tsx

```diff
@@ -96,88 +96,110 @@ const format = (date: Date) =>
-  return (
-    <>
-      <div>
-        <div className="min-w-4 mb-2">
-          <label htmlFor="availability" className="mt-0 flex text-sm font-medium text-neutral-700">
-            {t("availability")}
-          </label>
-        </div>
-        <Controller
-          name="schedule"
-          render={({ field }) => (
-            <AvailabilitySelect
-              value={field.value}
-              onBlur={field.onBlur}
-              name={field.name}
-              onChange={(selected) => {
-                field.onChange(selected?.value || null);
-              }}
-            />
-          )}
-        />
-      </div>
+  if (!isTeamEvent) {
+    return <EventTypeSchedule />;
+  }
 
-      <div className="space-y-4 rounded border p-4 py-6 pt-2 md:p-8">
-        <ol className="table border-collapse text-sm">
-          {weekdayNames(i18n.language, 1, "long").map((day, index) => {
-            const isAvailable = !!filterDays(index).length;
-            return (
-              <li key={day} className="my-6 flex border-transparent last:mb-2">
-                <span
-                  className={classNames(
-                    "w-20 font-medium sm:w-32",
-                    !isAvailable && "text-gray-500 opacity-50"
-                  )}>
-                  {day}
-                </span>
-                {isLoading ? (
-                  <SkeletonText className="block h-5 w-60" />
-                ) : isAvailable ? (
-                  <div className="space-y-3 text-right">
-                    {filterDays(index).map((dayRange, i) => (
-                      <div key={i} className="flex items-center leading-4">
-                        <span className="w-16 sm:w-28 sm:text-left">{format(dayRange.startTime)}</span>
-                        <span className="ml-4">-</span>
-                        <div className="ml-6">{format(dayRange.endTime)}</div>
-                      </div>
-                    ))}
-                  </div>
-                ) : (
-                  <span className="ml-6 text-gray-500 opacity-50 sm:ml-0">{t("unavailable")}</span>
-                )}
-              </li>
-            );
-          })}
-        </ol>
-        <hr />
-        <div className="flex flex-col justify-center gap-2 sm:flex-row sm:justify-between">
-          <span className="flex items-center justify-center text-sm text-gray-600 sm:justify-start">
-            <Icon.FiGlobe className="mr-2" />
-            {schedule?.timeZone || <SkeletonText className="block h-5 w-32" />}
-          </span>
-          <Button
-            href={`/availability/${schedule?.schedule.id}`}
-            color="minimal"
-            EndIcon={Icon.FiExternalLink}
-            target="_blank"
-            className="justify-center border sm:border-0"
-            rel="noopener noreferrer">
-            {t("edit_availability")}
-          </Button>
-        </div>
-      </div>
-    </>
+  return (
+    <Controller
+      name="metadata.useMemberSchedulesForTeamEvent"
```

Comment:
We want to keep `useMemberSchedulesForTeamEvent` as falsy by default because this has been the behaviour so far and thus it won't introduce any regression behaviour wise in existing team events.